.github/workflows/publish.yml

```diff
@@ -222,39 +222,6 @@ jobs:
       with:
         lane: build
         options: '{ "flavor": "itech_malawi" }'
-
-    - name: Unpack secrets cares_malawi_kch
```

Comment:
Did the jobs for `cares_malawi_kch`, `cares_malawi_mpc`, and `moh_togo_uat` get removed from there on purpose? If so, should we completely remove those brands from the repo?